### PR TITLE
Finished SQL inserts for static data : Caleb

### PR DESCRIPTION
**Pull request by CalebMathers**
For issue #75 

I've added SQL inserts for all of our static data so the schema file can be run to create the tables and populate them.

### CODE REVIEW

File: database/schema.sql [hunk 1]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
     PRIMARY KEY (image_id),
     FOREIGN KEY (region_id) REFERENCES region(region_id),
     FOREIGN KEY (constellation_id) REFERENCES constellation(constellation_id)
-);
\ No newline at end of file
+);
+
+INSERT INTO aurora_colour (aurora_colour_id, colour, description, meaning) VALUES
```
Comment (MurasatoNaoya) [inline database/schema.sql:118]: Great seed data and descriptions. We all looked at the AuroraWatch API and know that the UK can only be considered in one state at a given time, this expresses that well.

File: database/schema.sql [hunk 2]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
...
+(1, 'Green', 'No significant activity', 'Aurora is unlikely to be visible by eye or camera from anywhere in the UK.'),
+(2, 'Yellow', 'Minor geomagnetic activity', 'Aurora may be visible by eye from Scotland and may be visible by camera from Scotland, northern England and Northern Ireland.'),
+(3, 'Amber', 'Amber alert: possible aurora', 'Aurora is likely to be visible by eye from Scotland, northern England and Northern Ireland; possibly visible from elsewhere in the UK. Photographs of aurora are likely from anywhere in the UK.'),
+(4, 'Red', 'Red alert: aurora likely', 'It is likely that aurora will be visible by eye and camera from anywhere in the UK.');
+
+INSERT INTO region (region_name, latitude, longitude) VALUES
+('Northern Ireland', 54.61, -6.62),
```
Comment (MurasatoNaoya) [inline database/schema.sql:125]: Accurate long-lat values for all 12 regions we are considering. LGTM.

File: database/schema.sql [hunk 3]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
...
+('Yorkshire and the Humber', 53.80, -1.54),
+('East Midlands', 52.90, -1.23),
+('West Midlands', 52.48, -1.89),
+('East of England', 52.24, 0.90),
+('London', 51.51, -0.13),
+('South East', 51.28, -0.78),
+('South West', 50.78, -3.79);
+
+INSERT INTO county (county_name, latitude, longitude, region_id) VALUES
+('Antrim', 54.71, -6.22, 1),
```
Comment (MurasatoNaoya) [inline database/schema.sql:139]: Similar for all counties, this accuracy will be important for weather data later in the project.

File: database/schema.sql [hunk 4]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
...
+('Bristol', 51.45, -2.58, 12),
+('Cornwall', 50.45, -5.0, 12),
+('Devon', 50.71, -3.53, 12),
+('Dorset', 50.75, -2.44, 12),
+('Gloucestershire', 51.86, -2.24, 12),
+('Somerset', 51.1, -2.75, 12),
+('Wiltshire', 51.34, -1.99, 12);
+
+INSERT INTO constellation (constellation_name, constellation_short_name) VALUES
+('Andromeda', 'And'),
```
Comment (MurasatoNaoya) [inline database/schema.sql:258]: Abbreviations and necessary here, for the astronomyAPI. LGTM.

**Review by MurasatoNaoya — APPROVED** [review]

File: database/schema.sql [hunk 5]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
...
+
+('Berkshire', 51.45, -0.97, 11),
+('Buckinghamshire', 51.82, -0.83, 11),
+('East Sussex', 50.92, 0.27, 11),
+('Hampshire', 51.06, -1.31, 11),
+('Kent', 51.28, 0.52, 11),
+('Oxfordshire', 51.75, -1.26, 11),
+('Surrey', 51.25, -0.45, 11),
+('West Sussex', 50.91, -0.55, 11),
+
```
Comment (AlexOBarnes) [inline database/schema.sql:248]: Interesting use of linebreaks to separate regions

File: database/schema.sql [hunk 6]
```diff
@@ -113,4 +113,245 @@ CREATE TABLE image (
...
+INSERT INTO body (body_name, constellation_id) VALUES
+('Mercury', 86),
+('Venus', 49),
+('Mars', 38),
+('Jupiter', 78),
+('Saturn', 4),
+('Uranus', 78),
+('Neptune', 66),
+('Pluto', 4),
```
Comment (AlexOBarnes) [inline database/schema.sql:355]: Are we sure we are going to recognise Pluto as important?

**Review by AlexOBarnes — APPROVED** [review]
Looks good to me

File: database/schema.sql
Comment (olliejth) [inline database/schema.sql:1]: Very clearly laid out and easy to follow.

**Review by olliejth — APPROVED** [review]
Nice that we've got this out of the way, a good base to work on and build from.

**Pull request merged**